click/_winconsole.py

```diff
@@ -201,6 +201,30 @@ def __repr__(self):
         )
 
 
+class StreamWrapper(object):
```

Comment:
This should have a more descriptive name.

Reply:
Anything in mind? How about `WindowsConsoleStreamWrapper`?

Reply:
`WindowsChunkedWriter`?